container/image.bzl

```diff
@@ -168,6 +179,13 @@ def _image_config(
         args += ["--base=%s" % base_config.path]
         inputs += [base_config]
 
+    if base_manifest:
+      args += ["--basemanifest=%s" % base_manifest.path]
+      inputs += [base_manifest]
+
+    if operating_system:
+      args += ["--operating_system=%s" % operating_system]
+
```

Comment:
it seems you're not using the --architecture flag here

Reply:
I'm not, but it may be useful to plumb it all the way through for support for arm, etc.  What would you prefer?

Reply:
Looks like v2.MetadataTest doesn't understand architecture yet in the image config file.  I've manually verified that it's working, but it's not as good as a test.

I'll need to update the test project as well to make this work.  Would you like me to remove architecture altogether or bring another PR later with the test working?